test/test_torch.py

```diff
@@ -20411,9 +20465,9 @@ def test_split_view(self, device):
             v[0, 0] = idx + 1
             self.assertEqual(t[idx, 0], v[0, 0])
 
-    def test_movedim_view(self, device):
+    def _test_movedim_view(self, device, op):
```

Comment:
is this change relevant to `swapdims` & `swapaxes`?

Reply:
Not really relevant to `swapdims/swapaxes`. Was using it previously, but right now it only tests `movedim`. (Test is moved to test_transpose_view)

Have kept this particular change since it adds one more test case for `movedim`

Reply:
Rename this to something like `run_test` or `helper` and define it within `test_movedim_view`.